### DISCUSSION TITLE
What is the difference between normal vs Qt version?

### DISCUSSION THREAD

**Discussion by sfksuperman** [c0re100/qBittorrent-Enhanced-Edition#443] [Q&A]
I can see that there are 2 versions available:

```
qbittorrent_enhanced_4.5.0.10_x64_setup.exe
qbittorrent_enhanced_4.5.0.10_qt6_x64_setup.exe
```

I want to understand what is the difference between these 2 versions.

Thanks.

**Comment by slybunda** [comment]
qt6 needs windows 10 or newer.

**Reply by DOSforever** [reply]
> qt6 needs windows 10 or newer.

那是否意味着只要使用 Qt5  的就可以用在 Windows 7/8 上？比如最新版的 qbittorrent_enhanced_4.6.7.10_x64_setup.exe

Does that mean that anything that uses Qt5 will work on Windows 7/8? For example the latest version of qbittorrent_enhanced_4.6.7.10_x64_setup.exe
